### DISCUSSION TITLE
FAQs for reTerminal Usage | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1172] [Q&A]
# FAQs for reTerminal Usage | Seeed Studio Wiki

reTerminal-FAQ

https://wiki.seeedstudio.com/reTerminal-FAQ/

**Comment by wukoyu** [comment]
Why does the buzzer keep ringing when the power is turned on? Also, the ReTerminal APP on the desktop cannot be executed?

**Comment by j-broome** [comment]
The touchscreen orientation is perfect after running the seeed dt-overlays script, but when I use ANY form of remote desktop, the cursor orientation moves perpendicular to my actual mouse movements. I can't seem to find a solution or resolve this. Using VNC or Raspberry Pi Connect produce the same undesired mouse orientation. Tried adding a transformation matrix, but I don't know which "xinput" VNC is using or if it even affects it. Any help here would be much appreciated!
